textrender.h

```diff
@@ -96,7 +96,7 @@ class TextRender : public QQuickItem
     void setShowBufferScrollIndicator(bool s) { if(iShowBufferScrollIndicator!=s) { iShowBufferScrollIndicator=s; emit showBufferScrollIndicatorChanged(); } }
 
     Q_INVOKABLE QPointF cursorPixelPos();
-    Q_INVOKABLE QSizeF cellSize();
+    QSizeF cellSize();
```

Comment:
This is used still, in setTextRenderAttributes, so this will break won't it?

Reply:
Used there as property.